docs/controller-skills/military.md

```diff
@@ -55,11 +55,13 @@ Some Restricted Areas will be activated by NOTAM, most commonly for use by RAAF
 </figure>
 
 ### Aircraft operating inside Restricted Area
-Military aircraft are permitted to operate within their designated Restricted Area as per the relevant NOTAM. Even if this Restricted Area is inside your airspace, you have no control authority over it.
+Military aircraft are permitted to operate within their designated Restricted Area as per the relevant NOTAM. Even if this Restricted Area is inside your airspace, you have no control authority over it. If a AWEC aircraft is up and certified, you transfer all traffic to that aircraft. The AWEC aircraft will control up to the lateral boundries of the restricted airspace. AWEC aircraft will hand off departing traffic back to your control authority at 10NMs from the boundary.
```

Comment:
Could you please define what AWEC is, either on the abbreviations.md page, or within this page?

Reply:
Airborne Early Warning & Control aircraft controls the tactical battlespace, providing direction for fighter aircraft, surface combatants and land-based elements, as well as supporting aircraft such as tankers and intelligence platforms.

Reply:
Will put a piece into both pages